docs/synthetics/response-codes.md

```diff
@@ -0,0 +1,47 @@
+
+When you make a request to a website or API, the server sends back a response code that tells you whether the request was successful or if there was an issue.
+
+- **2xx: Successful Responses**
+These codes indicate that the request was successfully processed by the server.
+- **3xx: Redirection Responses**
+These codes indicate that the requested resource has been moved, and the client (e.g., a browser) needs to take additional action, such as following a redirect.
+- **4xx: Client Errors**
+These codes indicate that there was an issue with the request made by the client (the user or application making the request).
+- **5xx: Server Errors**
+These codes indicate that the server failed to process a valid request, usually due to an issue on the server side.
+
+## HTTP Monitors
+
+Within the single monitor screen, we’ve introduced a Response Codes report that shows the response codes returned from the endpoints during the selected time range.
+
+This report helps you identify when the monitored endpoint fails at specific times. For example, if an expensive database backup occurs every day at midnight and causes a website to return a 503 Gateway Timeout, the Response Codes report can help you spot failures that happen consistently around the same time, allowing you to take action accordingly.
+
+**What You Can See in the Report:**
+
+**Total number of response codes** within the selected time range.
+
+![Response Codes Numeric Component](../images/synthetics/response-codes-nc.png)
+
+**Response codes over time** for each selected location, helping you identify regular occurrences of failed responses at similar times.
+
+![Response Codes TimeSeries](../images/synthetics/response-codes-locations-timeseries.png)
+
+**Average response times by location,** allowing you to compare response codes against response times within the same time range.
+
+![Response Codes Response Times](../images/synthetics/response-codes-response-times.png)
+
+**Failed runs table** to help you dig deeper into the issues.
+
+![Response Codes Failed Runs](../images/synthetics/response-codes-failed-runs.png)
+
+## Browser Monitors
+
+In Browser Monitors, each monitor run includes multiple URL requests, and each URL can return different response codes. In the **Waterfall** tab, you will now see a filter option to filter out non-2xx response codes. When applied, this filter will only show URLs that returned a response code **other than 2xx.**
```

Comment:
In Browser Monitors, each monitor run includes multiple URL requests (e.g., to resources like images, JavaScript libraries, etc.)

Should we link "Waterfall" to https://sematext.com/docs/synthetics/browser-monitor/#waterfall-chart ?